content/copilot/github-copilot-chat/using-github-copilot-chat-in-your-ide.md

```diff
@@ -36,7 +36,7 @@ shortTitle: Using Copilot Chat
 
 {% data variables.product.prodname_copilot_chat %} is available to all organizations and enterprises that have an active {% data variables.product.prodname_copilot_for_business %} license. You can enable or disable {% data variables.product.prodname_copilot_chat %} for your organization or enterprise in the {% data variables.product.prodname_copilot_for_business %} settings page.
 
-If {% data variables.product.prodname_copilot_chat_short %} is enbaled or disabled at the enterprise level, organizations within the enterprise cannot override the setting.
+If {% data variables.product.prodname_copilot_chat_short %} is enabled or disabled at the enterprise level, organizations within the enterprise cannot override the setting.
```

Comment:
#### Before
https://docs.github.com/en/copilot/github-copilot-chat/using-github-copilot-chat-in-your-ide#enabling-or-disabling-github-copilot-chat
<img width="705" alt="image" src="https://github.com/github/docs/assets/2119212/70eaa839-7241-4601-95cb-627a075ef012">

#### After
https://docs-30890-94eb78.preview.ghdocs.com/en/copilot/github-copilot-chat/using-github-copilot-chat-in-your-ide#enabling-or-disabling-github-copilot-chat
<img width="704" alt="image" src="https://github.com/github/docs/assets/2119212/306d4b20-0abe-4ae2-ad3c-3c0053a185d8">